Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -565,4 +565,16 @@ extension AVFoundationPlayback {
         guard playbackType == .live else { return false }
         return seekableTimeRanges.first(where: { $0.timeRangeValue.duration.seconds >= minDvrSize}) != nil
     }
+
+    open override var dvrPosition: Double {
+        if let start = seekableTimeRanges.first?.timeRangeValue.start.seconds,
```

Comment:
Instead of use seekableTimeRanges.first? I think we need check all seekableTimeRanges and get the lower start value in order to guarantee the start seekable position.